browser/ui/views/playlist/playlist_add_bubble_view.cc

```diff
@@ -186,8 +187,9 @@ void PlaylistAddBubbleView::InitListView() {
   SetButtonEnabled(ui::DIALOG_BUTTON_OK, true);
   SetButtonEnabled(ui::DIALOG_BUTTON_CANCEL, true);
 
-  scroll_view_->SetPreferredSize(
-      gfx::Size(kWidth, scroll_view_->GetPreferredSize().height()));
+  scroll_view_->SetPreferredSize(gfx::Size(
```

Comment:
How about invalidate scroll view's previous preferred size by calling `scroll_view_->SetPreferredSize(std::nullopt);` instead of considering list view size?

I assume that we do this to have specific width(`kWidth`) for scroll view.
IMO, we should have specific width for dialog and then we only need to call `scroll_view_->SizeToPreferredSize();`.

Reply:
Well, I think Dialog's size should be decided automatically by preferred size of its children. I believe that's why `BubbleDialogDelegate::autosize_` and `BubbleDialogDelegate::SizeToContents()` exist.

Reply:
Got it.